### PR TITLE
Create wins.en.md

### PR DESCRIPTION
**Pull request by realjenny**

### CODE REVIEW

**Review by butterflyhug — APPROVED** [review]

File: frontend/content/power/wins.en.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+precedent for others with similar issues. But, our work together isn’t done. We need to ensure that what happened to Phares never again happens to any other worker.
+
+Read about our continued efforts to [fix the broken system](https://dropthedeadnames.org/).
+
+## Be bold interns
+
+We helped Google BOLD interns convince the company to give them a $5,000 stipend for housing and other needs during the COVID-19 pandemic. Read about
+[how Google interns fought for more cash—and won!](https://www.protocol.com/policy/google-interns-won-stipend)
+
+## Data center workers: know our rights
```
Comment (junefish) [inline frontend/content/power/wins.en.md:27]: should it be capitalised as "Know Our Rights"?

**Review by junefish — CHANGES_REQUESTED** [review]
see comment in-line

File: frontend/content/power/wins.en.md (hunk 1 above)
Comment (realjenny) [reply frontend/content/power/wins.en.md:27]: No, because the rest of the heading isn't capitalized. We use sentence case style not title case

Comment (junefish) [reply frontend/content/power/wins.en.md:27]: Okay!  I'd seen it in title case other places (e.g. emails) so wanted to check.

**Review by junefish — APPROVED** [review]

File: frontend/content/power/wins.en.md (hunk 1 above)
Comment (realjenny) [reply frontend/content/power/wins.en.md:27]: Appreciate the eagle eye! It's something that is changing in the world of grammar over time.

**Pull request merged**